UnicornRender/system/source/profiler/MouseProfiler.cpp

```diff
@@ -20,7 +20,7 @@ namespace system
 MouseProfiler::MouseProfiler(Manager& manager)
     : m_systemManager( manager )
 {
-    LOG_INFO("MouseProfiler created");
+    spdlog::get(utility::Logger::LoggerName())->info("MouseProfiler created");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.